dataikuapi/dss/ml.py

```diff
@@ -364,6 +364,14 @@ def __repr__(self):
     def set_values(self, values=None):
         self._algo_settings._set_categorical_values(self.name, values=values)
 
+    def enable_values(self, values_list):
+        for value in values_list:
+            self.set_values({value: {"enabled": True}})
+
+    def disable_values(self, values_list):
+        for value in values_list:
+            self.set_values({value: {"enabled": False}})
```

Comment:
Why not just `values` instead of `values_list`?

Reply:
It's just an old habit of mine of adding type to the name of variables, but I guess here we could accept any iterable and not only lists.